report_qweb_pdf_watermark/readme/HISTORY.rst

```diff
@@ -0,0 +1,4 @@
+12.0.1.0.0 (2019-11-18)
```

Comment:
this history file differ from the README.rst, not sure if there are some automation to sync those information ?

Reply:
Good catch! It's this file that needs to be updated, and then the README.rst will be regenerated automatically.